core/src/main/java/org/apache/iceberg/BaseMetastoreCatalog.java

```diff
@@ -36,6 +36,8 @@
 public abstract class BaseMetastoreCatalog implements Catalog {
   private static final Logger LOG = LoggerFactory.getLogger(BaseMetastoreCatalog.class);
 
+  protected static final String NAMESPACE_LOCATION_PROPS = "location";
```

Comment:
why not just keep this in the `NessieCatalog`?

Reply:
just put it back in the NessieCatalog